pkgs/top-level/all-packages.nix

```diff
@@ -29787,7 +29801,9 @@ with pkgs;
 
   waybar = callPackage ../applications/misc/waybar {};
 
-  waylock = callPackage ../applications/misc/waylock {};
+  waylock = callPackage ../applications/misc/waylock {
+    zig = zig_0_9_1;
```

Comment:
Each of these changes should be in a separate commit, documenting why this is needed.